src/components/EditCoursePage/EditCoursePage.test.jsx

```diff
@@ -467,6 +473,28 @@ describe('EditCoursePage', () => {
       expect(mockHandleCourseSubmit).toHaveBeenCalled();
     });
 
+    it('upon course submission, StatusAlert is set to appear', () => {
+      const component = shallow(<EditCoursePage {...
+              {
```

Comment:
nit: might be worth moving these props to a constant just for easy of modification down the road